src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -4233,8 +4233,56 @@
             </Trigger>
         </Style.Triggers>
     </Style>
-    
-    
+
+    <Style x:Key="OutlinedButtonStyleInverted" TargetType="Button">
```

Comment:
Would you add more comments about why this is a new style and currently used by which button?

Reply:
@QilongTang I've added extra comments here: https://github.com/DynamoDS/Dynamo/pull/12916/commits/6e3af78f9f27e36771c3ceb54e8ea240bf3c9047